PowerEditor/src/ScitillaComponent/FindReplaceDlg.cpp

```diff
@@ -1800,19 +1800,22 @@ void FindReplaceDlg::findAllIn(InWhat op)
 
 		_pFinder->_scintView.display();
 		_pFinder->display();
-		doSetMarkingStruct = true;
+		::SendMessage(_hParent, NPPM_DMMHIDE, 0, (LPARAM)_pFinder->getHSelf());
+		::UpdateWindow(_hParent);
+		justCreated = true;

```

Comment:
What's the point of `justCreated`?

Why not set it up at 1759, where it's first declared, *i.e.*:

```
const bool justCreated = (!_pFinder);
```

...is there some way that we create/allocate `_pFinder` but don't create it?